src/tests/control/test_auth.py

```diff
@@ -892,6 +893,19 @@ def test_log_out_after_relative_timeout_really_enforced(self):
         response = self.client.get('/control/')
         self.assertEqual(response.status_code, 302)
 
+    def test_plugin_auth_updates_auth_last_used(self):
+        session = self.client.session
+        session['pretix_auth_long_session'] = True
+        session['pretix_auth_login_time'] = int(time.time()) - 3600 * 5
+        session['pretix_auth_last_used'] = int(time.time()) - 3600 * 3 - 60
+        session.save()
+
+        request = RequestFactory().get("/")
+        request.session = self.client.session
+        process_login(request, self.user, keep_logged_in=True)
+
+        assert request.session['pretix_auth_last_used'] >= int(time.time())
```

Comment:
This only works "by accident" and is not stable. After all, this line is running *after* process_login, so we could already be in the next second if our computer is slow, or if process_login ran at a time with .999999 and we have bad luck. Let's allow some slack:

```suggestion
        assert request.session['pretix_auth_last_used'] >= int(time.time()) - 60
```

It still tests what the test is trying to do.

Reply:
Works for me, I added it to the test commit.